V5_MIGRATION.md

```diff
@@ -64,34 +64,22 @@ In v4, 3D Secure classes were housed in the `BraintreePaymentFlow` module. In v5
 
 #### Integration
 
-<details><summary>CocoaPods</summary>
-<p>
+##### CocoaPods
```

Comment:
We added the collapsable sections for each package manager because the SPM instructions were initially very long. Now that we only have a sentence of two for each, I think we can get rid of the collapsable sections.